databricks_cli/pipelines/cli.py

```diff
@@ -186,20 +187,24 @@ def list_cli(api_client):
 @provide_api_client
 def reset_cli(api_client, pipeline_id):
     """
+    (Deprecated) "reset" command is deprecated, please use "start --full-refresh" command instead.
```

Comment:
```suggestion
    (Deprecated) Use the "start --full-refresh" command.
```

I think we can make this short and to the point, but let me know what you think.

Also, a naive question: do we have a pattern for documenting deprecated commands? For example, it seems like elsewhere we still include the command description along with the recommended alternative. Does it make sense to include the deprecation admonition, but still include the command description below?

Reply:
And also, (Deprecated) or [Deprecated]? Here again, I'm not sure if there's a standard.

Reply:
I don't know about a standard. I was hoping our tech writer would know :)

Reply:
Looks like this is the only other place in the CLI where somethings deprecated:
https://github.com/databricks/databricks-cli/blob/25520f0b7a0ef8e37d318c50483a4f70393b6a08/databricks_cli/configure/provider.py#L129
```
def get_config_for_profile(profile):
    """
    [Deprecated] Reads from the filesystem and gets a DatabricksConfig for the
    specified profile. If it does not exist, then return a DatabricksConfig with fields set
    to None.
```
I agree with your suggestion of using square brackets.

Reply:
Changed to square brackets. Yeah, I couldn't find other examples, click.option has a deprecated param but the error message is generic and didn't have a easy way to modify that at the moment.

Let me know how the error messages look like now.

@jaseidman what do you mean by "deprecation admonition"?